packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset-kubeconfig/reset-kubeconfig.html

```diff
@@ -0,0 +1,38 @@
+<form
+    id="kubeClusterResetKubeconfig"
+    name="kubeClusterResetKubeconfig"
+    data-ng-submit="$ctrl.reset()"
+    novalidate
+>
+    <oui-modal
+        data-title="{{ ::'pci_projects_project_kubernetes_service_reset_kubeconfig' | translate }}"
+        data-primary-label="{{ ::'pci_projects_project_kubernetes_service_reset_kubeconfig_common_confirm' | translate }}"
+        data-primary-disabled="!$ctrl.isClusterReady"
+        data-secondary-action="$ctrl.goBack()"
+        data-secondary-label="{{ ::'pci_projects_project_kubernetes_service_reset_kubeconfig_common_cancel' | translate }}"
+        data-on-dismiss="$ctrl.goBack()"
+        data-loading="$ctrl.isReseting"
+    >
+        <div data-ng-if="$ctrl.isClusterReady">
+            <oui-message data-type="warning">
+                <span
+                    data-translate="pci_projects_project_kubernetes_service_reset_kubeconfig_message"
+                ></span>
+            </oui-message>
+            <p
+                data-translate="pci_projects_project_kubernetes_service_reset_kubeconfig_description1"
+            ></p>
+            <p
+                data-translate="pci_projects_project_kubernetes_service_reset_kubeconfig_description2"
+            ></p>
+            <p
+                data-translate="pci_projects_project_kubernetes_service_reset_kubeconfig_description3"
+            ></p>
+        </div>
+        <div data-ng-if="!$ctrl.isClusterReady">
```

Comment:
You remove `div` element as `p`is a block element.

Reply:
This is done